pkg/operator/operator.go

```diff
@@ -382,5 +384,23 @@ func (o *Operator) Config(key string) *manifests.Config {
 		glog.Warningf("Error loading proxy from API. Proceeding without it: %v", err)
 	}
 
+	s, err := o.client.GetSecret(o.namespace, prometheusEtcdCertSecretName)
+	if err != nil {
+		glog.Warningf("Error loading etcd certificates for Prometheus. Proceeding with etcd disabled. Error: %v", err)
```

Comment:
I guess we could do an early return here, but maybe this function grows in the future, so this is also fine with me.

Reply:
Agreed, I think the early return would cause future problems.